repos.yaml

```diff
@@ -1227,8 +1227,12 @@ repositories:
     maintainers:
     - name: CNIEG maintainers
       email: maintainers@cnieg.fr
+  - name: zhilyaev-marge
+    url: https://github.com/zhilyaev/marge-bot-helm
```

Comment:
Both URLs in both files should be the same. Can you please update this to be the same as the other URL.